### ISSUE TITLE
Is it possible to add 'size' to 'file' check?

### ISSUE THREAD

**Issue by taurus-forever** [aelsabbahy/goss#81]
Hi,

Right now to check the file size we have to write the second check based on ls/stat while it can be useful to have file size check as a part of 'file' attribute: https://github.com/aelsabbahy/goss/blob/master/docs/manual.md#attributes-1

Tnx!

**Comment by aelsabbahy** [comment]
I agree, this would be a great addition. But, it'll probably be a couple of weeks before I have the opportunity to look into this.

Any preference on units for size?

**Comment by taurus-forever** [comment]
speed: take your time, it is nice-to-have as workarounds are available here

units: even 'bytes' fine for me, having other options will force you please the game with proper supporting Kb/KB/KiB which is rudiment here, IMHO.

**Comment by aelsabbahy** [comment]
Fixed in [v0.1.7](https://github.com/aelsabbahy/goss/releases/tag/v0.1.7). Let me know if you run into any more issues.

**Comment by taurus-forever** [comment]
Works well, tnx!
```
me@host:~$ ./goss_0.1.6 v
Error: Invalid Attribute for File:/path/file.iso: size
me@host:~$ ./goss_0.1.7 v
...................................................................................................................................................

Total Duration: 0.115s
Count: 147, Failed: 0
me@host:~$ 
```
